src/Microsoft.ML.Transforms/Text/WrappedTextTransformers.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.TextAnalytics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.ML.Transforms
+{
+    /// <summary>
+    /// Word tokenizer splits text into tokens using the delimiter.
+    /// For each text input, the output column is a variable vector of text.
+    /// </summary>
+    public sealed class WordTokenizer : TrivialWrapperEstimator
+    {
+        /// <summary>
+        /// Tokenize incoming text in <paramref name="inputColumn"/> and output the tokens as <paramref name="outputColumn"/>.
+        /// </summary>
+        /// <param name="env">The environment.</param>
+        /// <param name="inputColumn">The column containing text to tokenize.</param>
+        /// <param name="outputColumn">The column containing output tokens. Null means <paramref name="inputColumn"/> is replaced.</param>
+        /// <param name="separators">The separators to use (comma separated).</param>
+        public WordTokenizer(IHostEnvironment env, string inputColumn, string outputColumn = null, string separators = "space")
```

Comment:
>string separators = "space" [](start = 99, length = 27)

This is a great thing to have as the arguments to a *command line*. To an API, we should use actual `char`.

Reply:
I started with `char`, then I discovered that multiple separators can be used.
I am all for making opportunistic changes to the transforms to make them friendlier and more API-appropriate, but only if this doesn't take too much extra time / lines of code. 

I also put a REVIEW somewhere saying exactly that.

---
In reply to: [218261276](https://github.com/dotnet/machinelearning/pull/931#discussion_r218261276) [](ancestors = 218261276)

Reply:
I would prefer to not block this PR on this issue though.

---
In reply to: [218288071](https://github.com/dotnet/machinelearning/pull/931#discussion_r218288071) [](ancestors = 218288071,218261276)

Reply:
I left a comment on line 55 and will create a GH issue

---
In reply to: [218289358](https://github.com/dotnet/machinelearning/pull/931#discussion_r218289358) [](ancestors = 218289358,218288153,218288071,218261276)